oem/ibm/service_files/pldm-create-phyp-nvram.service

```diff
@@ -1,10 +1,9 @@
 [Unit]
 Description=Create empty PHYP-NVRAM file
-ConditionPathExists=!/var/lib/pldm/PHYP-NVRAM
+ConditionPathExists=!/var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM
 
 [Service]
-ExecStart=/bin/mkdir -p /var/lib/pldm/
-ExecStart=/bin/dd if=/dev/zero of=/var/lib/pldm/PHYP-NVRAM bs=1024 count=145408
+ExecStart=/bin/sh -c "if [ -f /var/lib/pldm/PHYP-NVRAM ]; then mv /var/lib/pldm/PHYP-NVRAM /var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM; else dd if=/dev/zero of=/var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM bs=16 count=1; fi"
```

Comment:
why is the block size and count changed ?

Reply:
phyp cannot boot with 16bytes nvram , the original size was 142MB if i am not wrong.

Reply:
Fixed. I had wrongly copied the change from the cksum service file.